plume/src/main/kotlin/io/github/plume/oss/passes/graph/BaseCPGPass.kt

```diff
@@ -878,6 +879,28 @@ class BaseCPGPass(private val g: BriefUnitGraph) {
                 LocalCache.getType(local.type.toQuotedString())?.let { t -> builder.addEdge(this, t, EVAL_TYPE) }
             }
 
+    /**
+     * Creates a [NewIdentifier] from a [SootClass].
+     */
+    private fun createIdentifierVertex(
+        clazz: SootClass,
+        currentLine: Int,
+        currentCol: Int,
+        childIdx: Int = 1
+    ): NewIdentifierBuilder =
+        NewIdentifierBuilder()
+            .code(clazz.toString())
+            .name(clazz.toString())
+            .order(childIdx)
+            .argumentIndex(childIdx)
+            .typeFullName(clazz.type.toQuotedString())
+            .lineNumber(Option.apply(currentLine))
+            .columnNumber(Option.apply(currentCol))
+            .apply {
+//                LocalCache.getType(clazz.type.toQuotedString())?.let { t -> builder.addEdge(this, t, EVAL_TYPE) }
```

Comment:
You can uncomment this and then just +1 to the tests that check exact number of edges. Those tests are `UpdateGraphTest::[testFieldUpdateType|testClassUpdateModifier]` and `BasicIntraproceduralTest::basic5Test`